src/app.ts

```diff
@@ -2,19 +2,88 @@
 
 import Bugsnag from '@bugsnag/react-native'
 import { asObject, asString } from 'cleaners'
-import { Text, TextInput } from 'react-native'
+import { LogBox, Text, TextInput } from 'react-native'
 import RNFS from 'react-native-fs'
 
+import ENV from '../env.json'
 import { changeTheme, getTheme } from './components/services/ThemeContext'
-import { ENV } from './env'
 import { NumberMap } from './types/types'
 import { log, logToServer } from './util/logger'
 
+console.log = () => {}
+console.info = () => {}
+console.warn = () => {}
+
+const realConsDbg = console.debug
+
+type FgColor = 'b' | 'r' | 'g' | 'c' | 'm' | 'y'
+type BgColor = 'B' | 'R' | 'G' | 'C' | 'M' | 'Y'
+
+console.debug = (message?: any, color?: FgColor | BgColor | [FgColor, BgColor] | string) => {
+  const reset = '\x1b[0m'
+
+  const fgColors: Record<FgColor, string> = {
+    b: '\x1b[34m',
+    r: '\x1b[31m',
+    g: '\x1b[32m',
+    c: '\x1b[36m',
+    m: '\x1b[35m',
+    y: '\x1b[33m'
+  }
+
+  const bgColors: Record<BgColor, string> = {
+    B: '\x1b[44m',
+    R: '\x1b[41m',
+    G: '\x1b[42m',
+    C: '\x1b[46m',
+    M: '\x1b[45m',
+    Y: '\x1b[43m'
+  }
+
+  let foregroundColor = ''
+  let backgroundColor = ''
+
+  if (typeof color === 'string') {
+    if (color.length === 2) {
+      if (/[A-Z]/.test(color[0])) {
+        // 1st char is bg
+        backgroundColor = bgColors[color[0] as BgColor] ?? ''
+        foregroundColor = fgColors[color[1].toLowerCase() as FgColor] ?? ''
+      } else if (/[A-Z]/.test(color[1])) {
+        // 2nd char is bg
+        backgroundColor = bgColors[color[1] as BgColor] ?? ''
+        foregroundColor = fgColors[color[0].toLowerCase() as FgColor] ?? ''
+      }
+    } else if (color.length === 1) {
+      if (/[A-Z]/.test(color)) {
+        backgroundColor = bgColors[color.trim().toUpperCase() as BgColor] ?? ''
+      } else if (/[a-z]/.test(color)) {
+        foregroundColor = fgColors[color.trim().toLowerCase() as FgColor] ?? ''
+      }
+    }
+  }
+
+  if (!foregroundColor && !backgroundColor) {
+    foregroundColor = fgColors.y ?? ''
+  }
+
+  const deco = `${foregroundColor}${backgroundColor}`
+
+  const fmgMsg = typeof message === 'string' ? `'${message}${color != null ? `:${color}` : ''}'` : `${JSON.stringify({ message }, null, 2)}`
+
+  realConsDbg(`${deco}${fmgMsg}${reset}`)
+}
+
+console.debug('TEST   TEST')
+
 Bugsnag.start({
+  // @ts-expect-error
+  apiKey: ENV.BUGSNAG_API_KEY,
```

Comment:
Why wasn't this in place before and why the `@ts-expect-error`?

Reply:
Sorry this entire file wasn't supposed to end up in this review